components/coins-status/index.js

```diff
@@ -1,39 +1,66 @@
-import { useEffect } from 'react';
+import { useState, useEffect } from 'react';
 import { useAnimateValue } from './useCoinStatus';
 import styles from './coin.module.css';
 import PropTypes from 'prop-types';
 
+const BASE_API_URL = process.env.NEXT_PUBLIC_BASE_API_URL;
+const WALLET_URL = `${BASE_API_URL}/wallet`;
+
 const Coins = (props) => {
-  const [coins, changeCoins] = useAnimateValue(0, props.coin.value, 5000);
+  const { coin, balance } = props;
+  const [coins, changeCoins] = useAnimateValue(0, balance, 5000);
+
   useEffect(() => {
-    changeCoins(props.coin.value);
-  }, [props.coin.value, changeCoins]);
+    changeCoins(balance);
+  }, [balance, changeCoins]);
+
   return (
     <div className={styles.cointypeIndicator}>
       <div className={styles.coinData}>
-        <p> {props.coin.name}</p>
-        <p> {coins} </p>
+        <p> {coin}</p>
+        <p> {balance} </p>
       </div>
       <div className={`coin ${styles.rotateVertCenter}`}></div>
       <style jsx>{`
         .coin {
-          background-color: ${props.coin.color};
-          width: calc(4.25em / 2);
-          height: calc(4.25em / 2);
+          background-color: ${coin === 'neelam'
+            ? 'blue'
+            : coin === 'dinero'
+            ? 'green'
+            : 'gray'};
+          width: calc(5em / 2);
+          height: calc(5em / 2);
           border-radius: 50%;
           box-shadow: 3px 3px 3px #000;
-          border: 1px solid ${props.coin.borderColor};
+          border: 1px solid
+            ${coin === 'neelam' ? 'blue' : coin === 'dinero' ? 'green' : 'gray'};
```

Comment:
Nit: Can we reuse the conditional string here please?

Reply:
Done as below
```js
  const currencyColor = coin === CURRENCIES.NEELAM
    ? styles.blue
    : coin === CURRENCIES.DINERO
    ? styles.green
    : styles.gray;
```